.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+
+name: CI
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [run_build]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: ps2dev/ps2dev:latest
+    steps:
+    - name: Install dependencies
+      run: |
+        apk add build-base git zip
+    - uses: actions/checkout@v2
+    - run: |
+        git fetch --prune --unshallow
+
+
+    - name: Compile 
+      run: |
+        make
+    
+    - name: Get short SHA
+      id: slug
+      run: echo "::set-output name=sha8::$(echo ${GITHUB_SHA} | cut -c1-8)"
+
+    - name: Upload artifacts
+      if: ${{ success() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: MiniOPL-${{ steps.slug.outputs.sha8 }}
```

Comment:
I’d rather not call it miniOPL since it really is just an OPL launcher it doesn’t actually do anything other than get partition info and send it to OPL at boot, where as miniOPL really is a stripped down version of OPL